nixos/modules/services/networking/searx.nix

```diff
@@ -143,6 +144,35 @@ in
         '';
       };
 
+      faviconsSettings = mkOption {
+        type = types.attrsOf settingType;
+        default = { };
+        example = literalExpression ''
+          {
+            favicons = {
+              cfg_schema = 1;
+              cache = {
+                db_url = "/run/searx/faviconcache.db";
```

Comment:
This is not a good example. We should place it under ``/var/lib/searx``.

Reply:
I probably just carry it over to #346777 . The directory does not exist, so we need to probably add some systemd to option to create it.

Reply:
Ooh thank you !!!